test/functional/mining_basic.py

```diff
@@ -73,6 +76,42 @@ def mine_chain(self):
         self.restart_node(0)
         self.connect_nodes(0, 1)
 
+    def test_blockmintxfee_parameter(self):
+        self.log.info("Test -blockmintxfee setting")
+        self.restart_node(0, extra_args=['-minrelaytxfee=0'])
+        node = self.nodes[0]
+
+        # test default (no parameter), disabled (=0) and a bunch of arbitrary minimum tx fee rates [sat/kvB]
+        for blockmintxfee_sat_kvb in (DEFAULT_BLOCK_MIN_TX_FEE, 0, 50, 100, 500, 2500, 5000, 21000, 333333, 2500000):
+            blockmintxfee_btc_kvb = blockmintxfee_sat_kvb / Decimal(COIN)
+            if blockmintxfee_sat_kvb == DEFAULT_BLOCK_MIN_TX_FEE:
+                self.log.info(f"-> Default -blockmintxfee setting ({blockmintxfee_sat_kvb} sat/kvB)...")
+            else:
+                blockmintxfee_parameter = f"-blockmintxfee={blockmintxfee_btc_kvb:.8f}"
+                self.log.info(f"-> Test {blockmintxfee_parameter} ({blockmintxfee_sat_kvb} sat/kvB)...")
+                self.restart_node(0, extra_args=[blockmintxfee_parameter, '-minrelaytxfee=0'])
```

Comment:
Agree, done. Note that I had to trigger an UTXO set rescan for MiniWallet after restart, otherwise it would sometimes try to spend UTXOs from mempool transactions (which are now not available anymore).

Reply:
Ah good thinking!